app/src/main/res/layout/fragment_action_bar.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".ui.home.ActionBarFragment">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardView2"
```

Comment:
Tại sao không để CardView là ViewGroup to nhất mà phải để trong FrameLayout nhỉ?

Reply:
vì t chỉ muốn nó tác động đến action bar, thực chất vì ui này cũng chỉ có action bar nên việc đề ngoài hay để trong thì vẫn có tác dụng như nhau mà